src/earn/EarnCollectScreen.tsx

```diff
@@ -39,38 +39,54 @@ type Props = NativeStackScreenProps<StackParamList, Screens.EarnCollectScreen>
 export default function EarnCollectScreen({ route }: Props) {
   const { t } = useTranslation()
   const dispatch = useDispatch()
-  const { depositTokenId, poolTokenId } = route.params
-  const depositToken = useTokenInfo(depositTokenId)
-  const poolToken = useTokenInfo(poolTokenId)
+  const { pool } = route.params
+  const { depositTokenId } = pool.dataProps
   const withdrawStatus = useSelector(withdrawStatusSelector)
+  const positionsWithClaimableRewards = useSelector(positionsWithClaimableRewardsSelector)
```

Comment:
I think we may need to enable shortcuts for aave / allbridge for this to work. I assume these come from the `v2/getShorcuts` call

Reply:
Hmm, Do you think we should just create a new selector that isn't dependent on the shortcuts feature flag?